atc/db/worker_lifecycle.go

```diff
@@ -80,14 +80,7 @@ func (lifecycle *workerLifecycle) DeleteFinishedRetiringWorkers() ([]string, err
 		Join("containers c ON b.id = c.build_id").
 		Join("workers w ON w.name = c.worker_name").
 		LeftJoin("jobs j ON j.id = b.job_id").
-		Where(sq.Or{
-			sq.Eq{
-				"b.status": string(BuildStatusStarted),
-			},
-			sq.Eq{
-				"b.status": string(BuildStatusPending),
```

Comment:
Come to think of it I wonder if we ever needed to even check for pending builds. Why would a pending build have a container? :thinking: